packages/chromatic-api/chromatic-api.js

```diff
@@ -97,6 +95,38 @@ export const Chromatic = {
   },
 
   /**
+   * Adds style classes for an entry
+   * @param {String} type - the name of the entry
+   * @param {Array} styles - the css classes
+   * @returns {void}
+   */
+  addStyle(type, styles) {
```

Comment:
this is perfect, except the naming begs the question: are we mistakenly referring to a `style=` attribute with inline css? Perhaps this is really the `addClass` api?